src/mono/netcore/System.Private.CoreLib/src/System/Reflection/Emit/AQNBuilder.Mono.cs

```diff
@@ -0,0 +1,483 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+/*
+	This TypeNameBuilder is modified from CoreCLR's original.
+	It replaces the C++ bits of the implementation with a faithful C# port.
+	Original files:
+		src/coreclr/src/System.Private.CoreLib/src/System/Reflection/Emit/AQNBuilder.cs
+		src/coreclr/src/vm/typestring.cpp
+		src/coreclr/src/vm/typestring.h
+*/
+
+using System.Collections.Generic;
+using System.Collections;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace System.Reflection.Emit
+{
+	// TypeNameBuilder is NOT thread safe NOR reliable
+	internal class TypeNameBuilder
+	{
+		internal enum Format
```

Comment:
This file appears to be using tabs instead of spaces.  We've standardized on the latter for the rest of the repo.  Are we using tabs in the mono sources in general?